hw/ip/kmac/doc/_index.md

```diff
@@ -97,7 +97,7 @@ If the compile-time Verilog parameter `EnMasking` is not set, the first phase an
 
 If the masking feature is enabled, Keccak round logic stores the intermediate state after processing the 1st phase.
 The stored value are fed into the Chi function at the next cycle.
-Chi function implements [Domain-Oriented Masking (DOM) AND][]logic inside to protect against the 1st order SCA attacks.
+Chi function implements [Domain-Oriented Masking (DOM) AND][] logic inside to protect against the 1st order SCA attacks.
```

Comment:
nit: is "Domain-Oriented Masking (DOM) AND" supposed to be a link?

Reply:
sorry about, looks like the link works (must be completed elsewhere). marking resolved.